tests/phpunit/tests/test-switcher.php

```diff
@@ -186,4 +186,31 @@ function test_dropdown() {
 		$this->assertNotEmpty( $xpath->query( '//select/option[.="Français"]' )->length );
 		$this->assertNotEmpty( $xpath->query( '//script' )->length );
 	}
+
+	function test_with_hide_if_no_translation_option_in_admin_context() {
+		$links_model = self::$model->get_links_model();
+		$this->admin = new PLL_Admin( $links_model );
+		$this->admin->init();
+
+		$en = $this->factory->post->create();
+		self::$model->post->set_language( $en, 'en' );
+
+		$fr = $this->factory->post->create();
+		self::$model->post->set_language( $fr, 'fr' );
+
+		self::$model->clean_languages_cache(); // FIXME for some reason, I need to clear the cache to get an exact count
+
+		$args['hide_if_no_translation'] = 1;
+		$switcher = $this->switcher->the_languages( $this->admin->links, $args );
+
+		$doc = new DomDocument();
```

Comment:
May I suggest to add an `assertNotEmpty` assertion to test `$switcher` to make the test fail and not to an error

```
F                                                                   1 / 1 (100%)

Time: 9.58 seconds, Memory: 46.00MB

There was 1 failure:

1) Switcher_Test::test_with_hide_if_no_translation_option_in_admin_context
Failed asserting that a string is not empty.
```

instead of
```
E                                                                   1 / 1 (100%)

Time: 10.8 seconds, Memory: 46.00MB

There was 1 error:

1) Switcher_Test::test_with_hide_if_no_translation_option_in_admin_context
DOMDocument::loadHTML(): Empty string supplied as input
```